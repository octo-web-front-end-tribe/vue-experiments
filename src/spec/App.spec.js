import {shallow} from 'vue-test-utils';
import Vue from 'vue';
import Router from 'vue-router';

Vue.use(Router);

import App from '../App';

describe('App.vue', () => {
  it('should match snapshot', () => {
    // when
    const wrapper = shallow(App);

    // then
    expect(wrapper.element).toBeDefined();
    expect(wrapper.element).toMatchSnapshot();
  });
});
