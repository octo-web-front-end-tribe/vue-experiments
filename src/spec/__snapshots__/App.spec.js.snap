// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should match snapshot 1`] = `
<div
  id="app"
>
  <img
    src="./assets/logo.png"
  />
   
  <router-link
    exact=""
    to="/"
  >
    Home
  </router-link>
   
  <router-link
    to="counter"
  >
    Counter
  </router-link>
   
  <router-view />
</div>
`;
